README.rst

```diff
@@ -426,9 +426,30 @@ specified resource.  For example:
         pass
 
 The above view function will be called when either an HTTP POST or
-PUT is sent to ``/myview``.  In the next section we'll go over
-how you can introspect the given request in order to differentiate between
-various HTTP methods.
+PUT is sent to ``/myview``.
+
+Alternatively if you do not want to share the same view function across
```

Comment:
Sounds good. I can do that. Should I just add an additional section on multi HTTP method support in the `docs/source/topics/views.rst` that is a copy of this but is more thorough?